### PR TITLE
erhise/small-fixes

### PR DESCRIPTION
**Pull request by erhise**
* Removed running of offline-blitz from global npm command (gives error; and felt unnecessary)
* Added img to .npmignore to reduce package size
* Added some content to README.md

### CODE REVIEW

**Comment by siguse** [comment]
Looks great! 💥

**Comment by siguse** [comment]
Do we merge release-0.2.0 branch into master on release?

**Comment by erhise** [comment]
> Do we merge release-0.2.0 branch into master on release?

Yes, that's seems reasonable

File: README.md [hunk 1]
```diff
@@ -31,27 +26,72 @@ $ npm install -g offline-blitz
 
 # Run sandbox
 $ offline-blitz <filename>
+
+# Use with vim
+$ vim ...
```
[the remark this answers is not in the document]
Comment (erhise) [reply README.md:31]: Yes, you need to add this later

**Pull request merged**